v3/src/lib/handle-cfm-event.ts

```diff
@@ -1,14 +1,18 @@
+import { cloneDeep } from "lodash"
 import { CloudFileManagerClient, CloudFileManagerClientEvent } from "@concord-consortium/cloud-file-manager"
 import { appState } from "../models/app-state"
 import { removeDevUrlParams, urlParams } from "../utilities/url-params"
 import { wrapCfmCallback } from "./cfm-utils"
+import { DEBUG_CFM_EVENTS } from "./debug"
 
 import build from "../../build_number.json"
 import pkg from "../../package.json"
 
-export function handleCFMEvent(cfmClient: CloudFileManagerClient, event: CloudFileManagerClientEvent) {
-  // const { data, state, ...restEvent } = event
-  // console.log("cfmEventCallback", JSON.stringify({ ...restEvent }))
+export async function handleCFMEvent(cfmClient: CloudFileManagerClient, event: CloudFileManagerClientEvent) {
+  if (DEBUG_CFM_EVENTS) {
+    // We clone the event because the CFM reuses the same objects within the event
+    console.log("cfmEvent", event.type, cloneDeep(event))
```

Comment:
Nice addition for debugging/understanding how the CFM works. I expect you'll need to add an eslint-disable comment to avoid warnings in `lint:build`, though.